src/main/scala/com/microsoft/ml/spark/lightgbm/LightGBMClassifier.scala

```diff
@@ -155,6 +160,11 @@ class LightGBMClassificationModel(
       outputData = outputData.withColumn(getLeafPredictionCol,  predLeafUDF(col(getFeaturesCol)))
       numColsOutput += 1
     }
+    if (getFeaturesShapCol.nonEmpty) {
+      val featureShapUDF = udf(featuresShap _)
+      outputData = outputData.withColumn(getLeafPredictionCol,  featureShapUDF(col(getFeaturesCol)))
```

Comment:
should this be: getLeafPredictionCol -> getFeaturesShapCol (?)

Reply:
@imatiach-msft You're right. It is fixed now.